io.github.eminfedar.vaktisalah-gtk-rs.json

```diff
@@ -0,0 +1,86 @@
+{
+    "id": "io.github.eminfedar.vaktisalah-gtk-rs",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "vaktisalah-gtk-rs",
+    "finish-args": [
+        "--share=network",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        {
+            "name": "libsass",
+            "buildsystem":"meson",
+            "builddir": true,
+            "sources": [
+                {
+                    "type":"git",
+                    "url":"https://github.com/lazka/libsass.git",
+                    "commit": "aac79dccd3c8f7e8f22125f87a119f3b1ee9d487"
+                }
+            ]
+        },
+        {
+            "name": "sassc",
```

Comment:
isn't this also only for blueprint?

Reply:
Do you mean I should remove libsass & sassc and only blueprint & vaktisalah itself should remain?

Reply:
if it is only needed for blueprint it should have a similar cleanup rule too.

Reply:
so it's not even needed.